src/WorkItems/FilingClient.cs

```diff
@@ -17,14 +17,12 @@ public abstract class FilingClient
         /// The URI to the work item filing host.
         /// </summary>
         public Uri HostUri { get; internal set; }
-
         
         /// <summary>
         ///  The Azure DevOps account name or GitHub organization name.
```

Comment:
>Azure DevOps account name [](start = 17, length = 25)

Is this the full URI to the account or only the account name?

Reply:
Just the account name, now fixed, good catch.

---
In reply to: [381609466](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381609466) [](ancestors = 381609466)